2016/presentations/splash/text.md

```diff
@@ -0,0 +1,83 @@
+
+Title of Presentation
+=====================
+
+[Splash](http://splash.readthedocs.io/en/stable/) is an open source scriptable headless browser with 
+HTTP API written in Python 3 with Twisted & PyQt. I’d like to present the project and discuss 
+couple of social and technical phenomena that I noticed while taking part in development of 
+application. I’d like to discuss how application is structured, what libraries we used and
+how they served us, and finally discuss how open source works for us and how I perceive it.
+
+My presentation will have two parts, one technical and other "social/psychological". In the first 
+one I'll outline application architecture. In the second one I'll try to discuss some general
```

Comment:
unless you mean "application architecture" as a general topic, this should be "the application's architecture"